extensions/amp-carousel/amp-carousel-analytics.md

```diff
@@ -61,6 +61,14 @@ The `amp-carousel-next` event is issued when there is a travesal to the previous
 }
 ```
 
-### Vars
+## Carousel analytics variables
 
-Description of the variables can be found in the [analytics-vars.md](/extensions/amp-analytics/analytics-vars.md#fromslide) file.
+`<amp-carousel>` contributes the following variables, which can also be found in [analytics-vars.md](/extensions/amp-analytics/analytics-vars.md#fromslide).
+
+### fromSlide
```

Comment:
yup. This results in the documentation not getting updated in one place or the other. Would prefer to avoid this. If you think this is a better place for the documentation, please remove them from analytics-vars.md and add a link to this doc.